.travis.yml

```diff
@@ -22,6 +22,7 @@ install:
   - yarn install
 script:
   - yarn build
+  - yarn upgrade verovio-dev
```

Comment:
This seems to be unnecessary as travis should not cache an installed version of verovio or any other node packages. It is installed using the package present in the `/src` folder, so this seems redundant.